example/storybook/index.tsx

```diff
@@ -32,7 +32,7 @@ configure(() => {
 // To find allowed options for getStorybookUI
 const StorybookUIRoot = getStorybookUI({
   asyncStorage: null,
-  onDeviceUI: false,
+  onDeviceUI: true,
```

Comment:
Not sure if we want to commit this. I know @namick took it out because of style inconsistencies.

Reply:
I was curious why this was deactivated. I couldn't use pick any of the storybook stories without this.

Reply:
You can use `yarn web-controls` to pick stories. without the in simulator UI.  

I took it out because it wraps the stories in another view that adds padding which doesn't work for debugging the header/footers.

That said, I'm fine with adding it back.  It is easy to disable when needed.